dbms/src/Encryption/RateLimiter.cpp

```diff
@@ -897,4 +897,32 @@ IOLimitTuner::Watermark IOLimitTuner::getWatermark(int pct) const
         return Watermark::Low;
     }
 }
+
+IOLimitTuner::Watermark IOLimitTuner::getWatermark(const LimiterStatUPtr & fg, const LimiterStatUPtr & bg, int pct) const
+{
+    // There is a cornar case:
+    // 1. Both `fg_mbps` and `bg_mbps` are less than `min_bytes_per_sec`.
+    // 2. `bg` runs out of the quota, but `fg` has not been used.
+    // 3. In this case, `pct` can be less than `medium_pct` and watermark is LOW.
+    // 4. Low watermark means it is unnecessary to increase the quota by descrease the quota of others.
+    // 5. However, because `fg_mbps` is less than `min_bytes_per_sec`, `bg_mbps` cannot be increased by decreasing `fg_mbps`.
```

Comment:
Actually there are four similar corner case, it can hard to summary them,  maybe take one as an example is better?